src/main/java/wolox/training/models/Book.java

```diff
@@ -39,11 +42,15 @@
 	@Column(nullable = false)
 	private String isbn;
 
+	@ManyToOne
```

Comment:
So this implies that a Book can only be assigned to only one User? That seems a weird use case, if a Book is added to the collection of one user it could not be added by any other user after that. In the Trello card it specifies that a Book can be rented by multiple users.